R/simple_analysis.R

```diff
@@ -1,29 +1,30 @@
 #' Default tabulation
 #'
+#' This function is used when [analyze()] is invoked.
 #'
-#' This function is used when \code{\link{analyze}} is invoked
-#'
-#' @param x the \emph{already split} data being tabulated for a particular cell/set of cells
-#' @param \dots passed on directly
+#' @param x (`vector`)\cr the *already split* data being tabulated for a particular cell/set of cells.
+#' @param ... additional parameters to pass on.
 #'
 #' @details This function has the following behavior given particular types of inputs:
-#' \describe{
-#' \item{numeric}{calls \code{\link{mean}} on \code{x}}
-#' \item{logical}{calls \code{\link{sum}} on \code{x}}
-#' \item{factor}{calls \code{\link{length}} on \code{x}}
-#' }
+#'   \describe{
+#'     \item{numeric}{calls [mean()] on `x`.}
+#'     \item{logical}{calls [sum()] on `x`.}
+#'     \item{factor}{calls [length()] on `x`.}
+#'   }
 #'
-#' \code{in_rows} is called on the resulting value(s).
+#' The [in_rows()] function is called on the resulting value(s). All other classes of input currently lead to an error.
 #'
-#' All other classes of input currently lead to an error.
-#' @export
-#' @rdname rtinner
-#' @author Gabriel Becker and Adrian Waddell
 #' @inherit in_rows return
+#'
+#' @author Gabriel Becker and Adrian Waddell
+#'
 #' @examples
 #' simple_analysis(1:3)
 #' simple_analysis(iris$Species)
 #' simple_analysis(iris$Species == "setosa")
+#'
+#' @rdname rtinner
+#' @export
 setGeneric("simple_analysis", function(x, ...) standardGeneric("simple_analysis"))
```

Comment:
never heard of this one

Reply:
Does this have an entire file? We definitely should put this in utils and split files from some big ones like tt_export

Reply:
lets keep it for after release

Reply:
#852